apps/test-bundles/just.config.ts

```diff
@@ -1,4 +1,6 @@
 import { preset, task, resolveCwd } from '@fluentui/scripts';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore - parallel-webpack has no types
```

Comment:
I could write types manually for this package but I decided to not do so ATM (its the only usage in whole repo)